src/hooks/useInstallProgress.ts

```diff
@@ -0,0 +1,61 @@
+import { IpcRenderer, IpcRendererEvent } from 'electron'
+import { useEffect, useRef, useState } from 'react'
+import { InstallProgress, Runner } from '../types'
+
+const { ipcRenderer } = window.require('electron') as {
+  ipcRenderer: IpcRenderer
+}
+
+export function useInstallProgress(
```

Comment:
Great job with this hook. will be useful once we have a queue or a downloads screen